<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>161</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLCDNumber" name="metricsDisplay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>hell decoder</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hell_modeSelector">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>feldhell</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>slowhell</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hellx5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hellx9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fskhell</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fskh105</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fskh245</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hell80</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="hellText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>601</width>
      <height>71</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>599</width>
       <height>69</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="hell_switchBlackback">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>background</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hell_switchWidth">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="hell_rateCorrector">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
